vpr/src/place/grid_tile_lookup.h

```diff
@@ -31,11 +31,11 @@ class GridTileLookup {
 
     void fill_type_matrix(t_logical_block_type_ptr block_type, vtr::NdMatrix<int, 3>& type_count);
 
-    int region_tile_count(const Region& reg, t_logical_block_type_ptr block_type);
+    int region_tile_count(const Region& reg, t_logical_block_type_ptr block_type) const;
```

Comment:
Should move the comments from the .cpp file to .h file, and make Doxygen compatible too.